playbooks/common/openshift-cluster/config.yml

```diff
@@ -31,6 +31,6 @@
 
 - include: ../openshift-master/config.yml
 
-- include: additional_config.yml
-
 - include: ../openshift-node/config.yml
+
+- include: additional_config.yml
```

Comment:
Agreed on the refactoring, I just don't want to hold up this PR for that large of a change